### ISSUE TITLE
hyperapp version

### ISSUE THREAD

**Issue by kofifus** [jorgebucaran/hyperapp#1079]
Newbie here, two questions regarding versions:
-  the releases page (https://github.com/jorgebucaran/hyperapp/releases) tags 1.2.9 as "latest", does that mean 2.x is not production ready ? are there still breaking changes planned ?
- the same page shows the latest available as 2.0.0 but npm has 2.0.20, is there a changelog somewhere ?

Basically what version should a new project go for and where is that documented 

Thanks!

**Comment by zaceno** [comment]
New projects should use the latest 2.0 branch as the 1 branch has been deprecated for a few years now.

The 2.0.x has been stable for a while so I’d be very surprised if any breaking changes happen. But you’re right there hasn’t been an official announcement that it is “production ready” (although it is used in production in several places). 

As for documentation, there is no official website yet either but the docs folder in this repo is pretty well complete with all you need to know and more.

**Comment by kofifus** [comment]
Thank you, I think it will be good to have all versions after 2.0.0 included on  "releases"  and perhaps considering marking 2.0.0 or after as "stable"

**Comment by zaceno** [comment]
@kofifus I agree. Last I heard there were some plans to implement tests and then call it ready. (As docs and types are done).  Not exactly sure where we are with the tests but if it’s any comfort it’s been tested pretty extensively in the real world. Should be safe to call it stable and mark a release in my opinion.
